.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
+    ]
+}
```

Comment:
Do we really want to merge this?

Reply:
probably is better to just delete it right?

Reply:
I deleted the file